main.py

```diff
@@ -90,14 +79,29 @@ def main(core_context: det.core.Context, hparams: Dict[str, Any]):
         tasks=task_names,
         num_fewshot=args.num_fewshot,
         batch_size=args.batch_size,
+        max_batch_size=args.max_batch_size,
         device=args.device,
         no_cache=args.no_cache,
         limit=args.limit,
         description_dict=description_dict,
         check_integrity=args.check_integrity,
+        write_out=args.write_out,
+        output_base_path=args.output_base_path,
     )
     core_context.train.report_validation_metrics(steps_completed=0, metrics=results)
 
+    all_metrics = {}
+    assert len(results["results"]) == 1, "Each trial should execute one task only."
```

Comment:
Are you sure we want this check?  It mean the user can't use glob patterns like `hendrycks*`, IIUC

Reply:
ah, my bad. I didn't know about this use.